### PR TITLE
Added a LookupRevWork page, which views a revision work by lane, host, and revision name/hash.

### PR DESCRIPTION
**Pull request by ColonelThirtyTwo**
Needed for a script that my co-worker is working on; the data he is using has names but not IDs.

@rolfbjarne

### CODE REVIEW

**Comment by ColonelThirtyTwo** [comment]
Doesn't seem to be working; getting `Type MonkeyWrench.Web.UI.LookupRevWork not found.`

File: MonkeyWrench.Web.UI/LookupRevWork.ashx.cs [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+			using (var db = new DB ())
+			using (var cmd = db.CreateCommand (@"
+				SELECT revisionwork.host_id, revisionwork.lane_id, revisionwork.revision_id
+				FROM revisionwork
+				INNER JOIN host ON host.id = revisionwork.host_id
+				INNER JOIN lane ON lane.id = revisionwork.lane_id
+				INNER JOIN revision ON revision.id = revisionwork.revision_id
+				WHERE host = @host AND lane = @lane AND revision = @revision
+			")) {
+				DB.CreateParameter (cmd, "host", context.Request ["host"]);
```
Comment (rolfbjarne) [inline MonkeyWrench.Web.UI/LookupRevWork.ashx.cs]: You're not verifying the user here, this allows anyone to access.

**Comment by rolfbjarne** [comment]
ViewLane.aspx can already take named variables instead of the id:

`ViewLane.aspx?lane=Request["lane"]&host=Request["host"]&revision=["revision"]`

so this seems redundant.

**Comment by ColonelThirtyTwo** [comment]
Oh, I completely missed that ViewLane could take names instead of IDs. Whoops.

**Pull request closed without merging**